### PR TITLE
fix OCP-23044

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  please help to review, thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 1]
```diff
@@ -343,7 +343,7 @@ check_key_and_resource_link:
 check_key_and_action_link:
   element: &key_action
     selector:
-      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//button[text()='<text_in_action>'] | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1]//button[text()='<text_in_action>']
+      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1]//button | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1]//button
```
Comment (SSshahan) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: The text is not fixed, it will change as the total num of taints changing in my case.

Comment (yapei) [reply lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: Does the change affect `click_key_and_action_link`? If no, I think we can also remove the `text_in_action` param from feature files.

```
 click_key_and_action_link:
     element:
        <<: *key_action
        op: click
```

Comment (SSshahan) [reply lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: updated, thanks.

**Comment by SSshahan** [comment]
@akostadinov  @pruan-rht  could you help to merge it?

**Pull request merged**